src/sled_store.rs

```diff
@@ -72,8 +72,8 @@ impl StoreImpl for SledStore {
         Ok(value)
     }
 
-    /// Clear all key and value
-    /// clear also a set function so it will return SetError if Need
```

Comment:
music?

Reply:
😂aha,I just modified as you suggested earlier

Reply:
Haha, oops! That's what i get from trying to review on the phone!